src/main/java/bio/terra/service/ResourceService.java

```diff
@@ -0,0 +1,77 @@
+package bio.terra.service;
+
+import bio.terra.dao.ResourceDao;
+import bio.terra.metadata.BillingProfile;
+import bio.terra.metadata.MetadataEnumeration;
+import bio.terra.model.BillingProfileModel;
+import bio.terra.model.BillingProfileRequestModel;
+import bio.terra.model.DeleteResponseModel;
+import bio.terra.model.EnumerateBillingProfileModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Component
+public class ResourceService {
```

Comment:
Do you expect this service to grow to perform the resource management? It could, but it might be cleaner to separate the resource allocation service from the operations to manage billing accounts.

Reply:
Going to make a ProfileService